backend/entities/dataset_files/dataset_file.py

```diff
@@ -41,12 +40,18 @@ class DatasetFile:
     """
     df: pd.DataFrame
     categories: set
-    bias_calculator: BiasCalculator
+    score: float
+    category_scores: dict[str, float]
+    category_fprs: dict[str, dict[str, float]]
+    is_processed: bool
```

Comment:
Good extra metrics to store in the class so that they can be reused after being calculated once instead of being recalculated every time